Obsidian/Assets/tags.json

```diff
@@ -3219,7 +3277,7 @@
       "minecraft:flowing_lava"
     ]
   },
-  "fluids/water": {
+  "fluids/waters": {
```

Comment:
Uhh This might break tags in client would recommend changing them back 😅

Reply:
woops